### PR TITLE
[REVIEW] Bug ext python test

### LINKED ISSUE #80: [BUG] python tests: missing, fail, take very long, and generate warnings.
**cugraph/jaccard/test_jaccard.py**
Aborted (core dumped)

**cugraph/grmat & cugraph/nvgraph**
no test

**cugraph/luvain, cugraph/pagerank, & cugraph/sssp**
deprecated API warning

**cugraph/graph/test_graph.py**
take very long (#83)

====================================== warnings summary =======================================
/home/seunghwak/Program/anaconda3/envs/cugraph_dev/lib/python3.7/site-packages/yaml/constructor.py:126: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
if not isinstance(key, collections.Hashable):

-- Docs: https://docs.pytest.org/en/latest/warnings.html

### PR DESCRIPTION
**Pull request by seunghwak**
closes #80, #98

### CODE REVIEW

File: python/cugraph/graph/c_graph.pyx [hunk 1]
```diff
@@ -100,18 +118,24 @@ class Graph:
...
-        cudf.bindings.cudf_cpp.check_gdf_error(err) 
-        
+        try:
+            err = gdf_edge_list_view(< gdf_graph *> graph,
+                                     < gdf_column *> source,
+                                     < gdf_column *> dest,
+                                     < gdf_column *> value)
+            cudf.bindings.cudf_cpp.check_gdf_error(err)
+        finally:
+            delete_column(source)
```
[the remark this answers is not in the document]
Comment (seunghwak) [reply python/cugraph/graph/c_graph.pyx]: @jwyles Thank you very much for the comment!!!

This is not deleting data, this is deleting memory allocated by

cdef gdf_column * c_col = < gdf_column *> malloc(sizeof(gdf_column)) in create_column

c_col is used only temporarily to pass the parameters set in gdf_column_view_augmented in create_column.

c_col (sizeof(gdf_column), not the actual data columns) needs to be freed.

File: python/cugraph/graph/c_graph.pyx [hunk 2]
```diff
@@ -159,22 +188,28 @@ class Graph:
...
-        cudf.bindings.cudf_cpp.check_gdf_error(err)
+
+        try:
+            err = gdf_adj_list_view(< gdf_graph *> graph,
+                                    < gdf_column *> offsets,
+                                    < gdf_column *> indices,
+                                    < gdf_column *> value)
+            cudf.bindings.cudf_cpp.check_gdf_error(err)
+        finally:
+            delete_column(offsets)
```
[the remark this answers is not in the document]
Comment (seunghwak) [reply python/cugraph/graph/c_graph.pyx]: This is same, this is not deleting the actual data, deleting a temporary gdf_column instance used to pass parameters.

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,11 +21,13 @@
 - PR #95 Code clean up
 - PR #96 Relocated mmio.c and mmio.h (external files) to thirdparty/mmio
 - PR #97 Updated python tests to speed them up
+- PR #105 Updated ptyhton code to follow PEP8 (fixed flake8 complaints)
```
[the remark this answers is not in the document]
Comment (seunghwak) [reply CHANGELOG.md]: Thanks, I will fix this!!!

File: python/cugraph/graph/c_graph.pyx (hunk 1 above)
[the remark this answers is not in the document]
Comment (seunghwak) [reply python/cugraph/graph/c_graph.pyx]: Please note that create_column does not allocate memory to store actual data, it just creates a gdf_column instance (by malloc(sizeof(gdf_column))) and sets its attributes without actually allocating memory. delete_column mirrors this and just frees memory of the size sizeof(gdf_column).

[the remark this answers is not in the document]
Comment (jwyles) [reply python/cugraph/graph/c_graph.pyx]: Resolved.

File: python/cugraph/graph/c_graph.pyx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jwyles) [reply python/cugraph/graph/c_graph.pyx]: Resolved.

**Comment by BradReesWork** [comment]
rerun tests

**Pull request merged**